ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -1,6 +1,21 @@
 {# eos - SNMP Settings#}
 {% if snmp_server is arista.avd.defined %}
 !
+{%     if snmp_server.engine_ids is arista.avd.defined %}
+{%         if snmp_server.engine_ids.local is arista.avd.defined %}
+snmp-server engineID local {{ snmp_server.engine_ids.local }}
+{%         endif %}
```

Comment:
```suggestion
{%     if snmp_server.engine_ids.local is arista.avd.defined %}
snmp-server engineID local {{ snmp_server.engine_ids.local }}
{%     endif %}
```

Reply:
To match the cli output we need to place the ```snmp_server.engine_ids.remotes``` after line 143. So I suggested the above change for lines 4-7.

FYI:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/88734745/166814039-13d5e76a-cd99-4f24-8499-fd33d0ac5fdf.png">

Reply:
I assume you mean line 14 here right? but yeah good catch thanks!